server/graphql/common/expenses.ts

```diff
@@ -1502,6 +1556,44 @@ export async function editExpenseDraft(req: express.Request, expenseData: Expens
   });
 }
 
+/**
+ * A simple helper to handle the case when editing only the tags and/or accounting category of an expense.
+ * Permissions for these fields (especially regarding the status of the expense) are more relaxed than for other fields.
+ */
+const editOnlyTagsAndAccountingCategory = async (
```

Comment:
Nice! I like this, especially due to the complexity of editing an expense. Perhaps we should have a separate mutation to edit this type of meta-information about expenses and orders.